recipes/uncrustify/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conan.tools.files import rename
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class UncrustifyConan(ConanFile):
+    name = "uncrustify"
+    description = "Code beautifier"
+    license = "GPL2"
```

Comment:
```suggestion
    license = "GPL-2.0-or-later"
```

https://spdx.org/licenses/GPL-2.0-or-later.html